ARCHITECTURE.md

```diff
@@ -188,3 +198,34 @@ kURL automation, more specifically the `import-external-addons` GitHub action, p
 New versions are published to the external add-on registry and packages are copied from the source and stored in the kURL S3 bucket.
 
 The kURL API merges the external add-on registry with its internal list of add-on versions, making them available to the end-user.
+
+## Deployment and Releases
+
+Upon releasing kURL, scripts and add-on package archives are built and uploaded to to the kURL object storage bucket along with metadata including the Git sha from which they were generated.
+Once complete, the VERSION file is updated to point to the current version of kURL.
```

Comment:
To understand how it works would be great know what is the format of this VERSION file?
What has in this Version file? Is possible to provide an example?

Reply:
I've linked to the file itself